src/Recipe.php

```diff
@@ -70,6 +70,32 @@ public function isContrib(): bool
         return $this->data['is_contrib'] ?? false;
     }
 
+    public static function queueCompare(Recipe $first, Recipe $second): int
+    {
+        static $order = ['symfony/flex' => -2, 'symfony/framework-bundle' => -1];
+
+        $weight = [0, 0];
+        /** @var Recipe[] $pair */
+        $pair = [$first, $second];
+        for ($i=0; $i<2; $i++) {
+            $name = $pair[$i]->getName();
+            if (!isset($order[$name])) {
+                $weight[$i] = 0;
+            } elseif ($pair[$i]->getJob() == 'uninstall') {
+                $weight[$i] = -$order[$name];
+            } else {
+                $weight[$i] = $order[$name];
+            }
+        }
+        if ($weight[0] == $weight[1]) {
+            return 0;
+        } elseif ($weight[0] <= $weight[1]) {
+            return -1;
+        } else {
+            return 1;
+        }
```

Comment:
Isn't this a good use-case for the spaceship operator? 😄

Reply:
I haven't used it before and forgot about it! =) I'll use it